manager/allocator/allocator.go

```diff
@@ -156,7 +156,6 @@ func (a *Allocator) init(ctx context.Context, aa *allocActor) (<-chan events.Eve
 		api.EventDeleteNetwork{},
 		api.EventCreateService{},
 		api.EventUpdateService{},
-		api.EventDeleteService{},
```

Comment:
How was the allocator previously using this event ?

Reply:
I see that the change to not act on EventDeleteService in allocator/network.go is in a different commit. I'd suggest keeping it in the same commit. If you're planning to merge them, please ignore my comment.